packages/wonder-blocks-modal/components/modal-launcher.js

```diff
@@ -65,6 +67,16 @@ type State = {|
  * the `modal` prop.
  */
 export default class ModalLauncher extends React.Component<Props, State> {
+    static getDerivedStateFromProps(props: Props, state: State) {
+        return {
+            opened: props.opened != null ? props.opened : state.opened,
```

Comment:
I think a slightly better check here would be: `typeof props.opened === "boolean" ? props.opened : state.opened;`